src/cadenza.js

```diff
@@ -145,15 +145,21 @@ export class CadenzaClient {
    * @param {object} [options]
    * @param {boolean} [options.hideMainHeaderAndFooter] - Whether to hide the main Cadenza header and footer.
    * @param {boolean} [options.hideWorkbookToolBar] - Whether to hide the workbook toolbar.
+   * @param {string} [options.accept] - Media type; set to "application/pdf" for Jasper Report views
+   *     to show the PDF directly, without any Cadenza chrome around it.
```

Comment:
Although "chrome" is technically correct term, it always causes questions. I'd go with the words from the example "without any Cadenza footers and headers".

Reply:
Right, that's less ambiguous :)